src/core/core.cpp

```diff
@@ -341,7 +341,8 @@ void Core::start()
     }
     else
     {
-        qDebug() << "Self avatar not found";
+        qDebug() << "Self avatar not found, will broadcast empty avatar to friends";
+        setAvatar(nullptr);
```

Comment:
Nope. You can't just pass a nullptr to a function that expects a reference, that's all kinds of **UNDEFINED BEHAVIOR** right there.

You probably want to pass an empty QByteArray instead:
```
setAvatar(QByteArray()); // Verbose way
setAvatar({}); // Unreadable way
```